### PR TITLE
create quickselect.kt

### PR DESCRIPTION
**Pull request by Awani-Shukla**
this is kotlin algorithm for quick select.

**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/selection_algorithms/src/quick_select.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        return quickSelect(list, left, pivotIndex - 1, k)
+    else
+        return quickSelect(list, pivotIndex + 1, right, k)
+}
+ 
+fun main(args: Array<String>) {
+    val list = intArrayOf(9, 8, 7, 6, 5, 0, 1, 2, 3, 4)
+    val right = list.size - 1
+    for (k in 0..9) {
+        print(quickSelect(list, 0, right, k))
```
Comment (abdouskamel) [inline code/selection_algorithms/src/quick_select.kt]: Why do you execute quickselect nine times ?

File: code/selection_algorithms/src/quick_select.kt [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const val MAX = Int.MAX_VALUE
```
Comment (abdouskamel) [inline code/selection_algorithms/src/quick_select.kt]: You should put more empty lines in your code.

File: code/selection_algorithms/src/quick_select.kt (hunk 1 above)
Comment (Awani-Shukla) [reply code/selection_algorithms/src/quick_select.kt]: array length is 10 so thats why I called quickselect function 9 times

File: code/selection_algorithms/src/quick_select.kt (hunk 2 above)
Comment (Awani-Shukla) [reply code/selection_algorithms/src/quick_select.kt]: yeah, i make sure that I will work in accordance with the information provided by guide on style of writing codes.

File: code/selection_algorithms/src/quick_select.kt (hunk 1 above)
Comment (abdouskamel) [reply code/selection_algorithms/src/quick_select.kt]: I thought it was quicksort, i'm sorry.

Comment (Awani-Shukla) [reply code/selection_algorithms/src/quick_select.kt]: No problem it happens.

File: code/selection_algorithms/src/quick_select.kt (hunk 2 above)
Comment (Awani-Shukla) [reply code/selection_algorithms/src/quick_select.kt]: @abdouskamel I have updated this file have a loook.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**